RFS/test-resources/inventory.md

```diff
@@ -426,12 +426,36 @@ curl -X PUT "localhost:19200/_index_template/fwc_template" -H "Content-Type: app
 
 curl -X PUT "localhost:19200/fwc_index_1" -H "Content-Type: application/json"
 
-curl -X PUT "localhost:19200/fwc_alias/_doc/bwc_doc" -H "Content-Type: application/json" -d '
+curl -X PUT "localhost:19200/fwc_alias/_doc/fwc_doc" -H "Content-Type: application/json" -d '
 {
   "title": "This is a doc in a forward compatible index",
   "content": "Life, the Universe, and Everything"
 }'
 
+curl -X PUT "localhost:19200/no_mappings_no_docs" -H "Content-Type: application/json" -d '
+{
+  "settings": {
+    "index": {
+      "number_of_shards": 1,
+      "number_of_replicas": 0
+    }
+  }
+}'
+
+curl -X PUT "localhost:19200/empty_mappings_no_docs" -H "Content-Type: application/json" -d '
+{
+  "settings": {
+    "index": {
+      "number_of_shards": 1,
+      "number_of_replicas": 0
+    }
+  },
+  "mappings": {
+    "properties":{
+    }
+  }
+}'
```

Comment:
I'd love to see us generate these on-the-fly and then cache them like we do for preloaded data images.  In fact, use the preloaded images, then just take a snapshot via a helper program.  Make the output of that something that gets cached as a build artifact and then it can become a dependency to the tests.  It seems like that could be 1-2 hours of work and removes the risks of many hours of work trying to figure out if there was a mismatch later on.